.gitignore

```diff
@@ -37,4 +37,4 @@ _trial_temp*/
 /docs/build/
 /htmlcov
 /pip-wheel-metadata/
-
+logs/
```

Comment:
I think we should just get rid of this, but if you think it is important, can you (a) anchor it and (b) sort it alphabetically.

Reply:
done